### DISCUSSION TITLE
focus output next

### DISCUSSION THREAD

**Discussion by arcmags** [i3/i3#5125] [Q&A] [answered]
One can move a workspace to the next output with:
    
    move workspace to output next

But `output next` does not work with `focus`.  As a workaround, right now, I use:

    focus output right, focus output down

This is undesirable, though, as it doesn't work properly with multiple outputs in a mix of both horizontal and vertical relative positions.  It would be nice to simply have:

    focus output next

**Comment by vincentbernat** [comment, marked as answer]
This is https://github.com/i3/i3/pull/4622. It will be available in 4.21 (soon to be released).
